torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -595,29 +595,10 @@ def rewrite_node(node):
                 return None
 
             if node.target == torch.ops.aten.div.Tensor:
-                # ttnn.recip does not support scalars. Call an ttnn.full and pass that to ttnn.recip
-                # TODO(kevinwuTT): Use a ttnn equivalent
-                node_metadata = node.meta["val"]
-                shape = node_metadata.size()
-                # If last dim == 1, then the follow error will appear:
-                # Page size must be divisible by sizeof(uint32_t) because buffers hold uint32_t values
-                if shape[-1] != 1 and HasValidPageSize(shape):
-                    if isinstance(args[1], float):
-                        new_kwargs = {
-                            "fill_value": args[1],
-                            "device": TtnnDevice(),
-                        }
-                        full = g.call_function(
-                            ttnn.full,
-                            args=(tuple(shape),),
-                            kwargs=new_kwargs,
-                        )
-                        to_layout = g.call_function(ttnn.to_layout, (full,), {"layout": TtnnTileLayout()})
-                        recip = g.call_function(ttnn.reciprocal, (to_layout,), {})
-                    else:
-                        recip = g.call_function(ttnn.reciprocal, (args[1],), {})
+                if not isinstance(args[1], float) and (get_shape(args[0]) != get_shape(args[1])):
```

Comment:
Please fire a ticket and link to https://github.com/tenstorrent/pytorch2.0_ttnn/issues/479

Reply:
Yeah, I made the ticket here: https://github.com/tenstorrent/tt-metal/issues/15445